Twig/Extension/SonataAdminExtension.php

```diff
@@ -171,6 +174,8 @@ public function renderListElement(
     }
 
     /**
+     * @internal
```

Comment:
If this method if public only for tests, then we have a conception problem IMHO.

Could we instead try to change the test to be able to make this private?